src/rules/selector-nest-combinators/index.js

```diff
@@ -20,21 +21,39 @@ export default function(expectation) {
       return;
     }
 
+    function precedesParentSelector(current) {
+      do {
+        current = current.next();
+
+        if (current.type === "nesting") {
+          return true;
+        }
+      } while (current.next());
+
+      return false;
+    }
+
+    // attribute, class, combinator, comment, id, nesting, pseudo, root, selector, string, tag, or universal
+    const chainingTypes = [
+      "attribute",
+      "class",
+      "id",
+      "pseudo",
+      "tag",
+      "universal"
+    ];
+
+    const interpolationRe = /#{.+}$/;
```

Comment:
Given that this is checking for the presence of any interpolation in the current selector, it's probably not particularly relevant, but it would generallly be better to use the non-greedy + operator? So that it would not match `#{$var} .selector li #{$var-2}`. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp#quantifiers